src/client/components/AdditionalProductCard.tsx

```diff
@@ -42,38 +41,33 @@ export const AdditionalProductCard = ({
   )
 }
 
-const commonStyles = css({
-  all: 'unset',
-  cursor: 'pointer',
-  position: 'relative',
-  display: 'flex',
-  minHeight: '8rem',
-  borderRadius: 8,
-
-  overflow: 'hidden',
-  backgroundColor: colorsV3.white,
-  transition: 'all 150ms',
-  '&:hover:not([disabled]), &:focus-visible': {
-    boxShadow: `0 0 0 1px ${colorsV3.black}, 0px 2px 2px rgba(0, 0, 0, 0.1)`,
-
-    // otherwise text in the LinkCard will turn purple on hover since they're inside an anchor element
-    color: 'initial',
-  },
-
-  '&:disabled': {
-    opacity: 0.5,
-  },
-})
-
 export const Card = styled.button<Pick<AdditionalProductCardProps, 'checked'>>(
   ({ checked }) => ({
-    ...commonStyles,
```

Comment:
For some reason I couldn't figure it out, `...commonStyles` was invalidating all the styles defined after.